sdk/search/azure-search/src/main/java/com/azure/search/models/IndexBatchImpl.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/**
+ * Contains a batch of document write actions to send to the index.
+ */
+@Fluent
+public class IndexBatchImpl<T> {
```

Comment:
Since this class is generated by autorest, this change should also be updated in the swagger config. Please advice on the final naming decision to open a seperate PR in the spec repo. Once it is merged the code can be re-generated

Reply:
@navalev This is called `IndexBatch` in the Swagger I believe, so there must be a rewrite rule in the config we need to change, right? We can do that after merging this PR.

Reply:
yes, the name is changed in autorest config. I was meaning that any change requires re-generating the code